app-misc/hyfetch/hyfetch-9999.ebuild

```diff
@@ -20,10 +21,27 @@ fi
 LICENSE="MIT"
 SLOT="0"
 
+PATCHES=(
+	"${FILESDIR}"/${PN}-1.4.11-pyproject.patch
+	"${FILESDIR}"/${PN}-1.4.11-neofetch.patch
+)
```

Comment:
Are there any plans to upstream these patches? Since you could just commit them there. Then we shouldn't include these patches at all in the -9999 ebuild.

Reply:
The neofetch patch I don't intend to, since one of the non-package manager install methods would be broken by it. But converting to pyproject I do plan to make a PR for soon.